robusta_krr/core/integrations/prometheus/metrics_service/victoria_metrics_service.py

```diff
@@ -26,20 +27,13 @@ def find_metrics_url(self, *, api_client: Optional[ApiClient] = None) -> Optiona
         )
 
 
-class VictoriaMetricsNotFound(MetricsNotFound):
-    """
-    An exception raised when Victoria Metrics is not found.
-    """
-
-    pass
-
-
 class VictoriaMetricsService(PrometheusMetricsService):
     """
     A class for fetching metrics from Victoria Metrics.
     """
 
     service_discovery = VictoriaMetricsDiscovery
+    is_victoria_metrics: bool = True
```

Comment:
What is this needed for? You can also do `isinstance(metric_service, VictoriaMetricsService)`